pages/Documents/ConversationalAI/Third-PartyBots/MicrosoftDirectLine/basic-content.md

```diff
@@ -66,18 +68,18 @@ Figure 2.1 Activity with Structured Content
 <br>
 <br>
 
-
 ### Sending Quick Replies
 
 {: .important}
 Quick Replies are only supported in Messaging Conversations.
 
-Quick Replies are a special type of structured content. It is a message sent along with predefined answers. The 
-documentation can be found [here](quick-replies-introduction-to-quick-replies.html).
-For detailed information on Quick Replies check out the documentation for the respective channel 
-([Mobile SDK and Web](mobile-sdk-and-web-templates-quick-replies-template.html), 
-[Facebook Messenger](facebook-messenger-templates-quick-replies-template.html), 
-[Google RCS Business Messaging](google-rcs-business-messaging-templates-quick-replies-template.html))
+Quick Replies are a special kind of the Structured Content.
+The content should be added to the quickReplies property of the structuredContent object and there also
```

Comment:
there **also** should **be** a message included

Or a general overhaul :
The content needs to be provided as a quickReplies object in the structuredContent. It is also required to provide a message property in the structuredContent,

Reply:
Was this the only content change for Microsoft?  The rest seems to be removal of whitespace? Or did you remove line breaks? Can't pull your repo to check myself

Reply:
Ok it looks like you just removed the spaces. I think i kept them in originally when i added the softwraps because i could just remove the softwrap without the need to add a space again. kramdown renders a space and a softwrap still just as one space. But if you prefer removing it that's fine

Reply:
Yep there was a dead link in the old one thus removed it and made the text better